frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedTab.test.tsx

```diff
@@ -3,8 +3,8 @@ import 'leaflet/dist/leaflet.css';
 
```

Comment:
we don't have parcels with associated buildings, or buildings with associated parcels, so just delete this.